### ISSUE TITLE
Vibration for pop up folder

### ISSUE THREAD

**Issue by hadyanfadlillah** [8bitPit/Niagara-Issues#2939]
### Describe the solution you'd like

Please add vibration when pop up folder opens.
Thank you.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by NiagaraSupport** [comment]
Hey, thanks for the request.
Unfortunately, haptic feedback is usually system-specific. For example, feedback on pop-up folders worked fine on a Redmi Note 9 (just tested it). Could you share what device you are using?

**Comment by hadyanfadlillah** [comment]
Yes, my Samsung phone has the haptic feedback. However on my Iqoo Z9 there is no vibration on folder pop ups
